ui/src/criteria/concept.tsx

```diff
@@ -365,29 +374,12 @@ function ConceptEdit(props: ConceptEditProps) {
           }}
         />
       </Loading>
-    </>
+    </Box>
   );
 }
 
-type ConceptInlineProps = {
-  data: Data;
-};
-
-function ConceptInline(props: ConceptInlineProps) {
```

Comment:
Yes, though there's currently nothing to actually edit there. Eventually that's where modifiers will go. Once I removed the current content I ended up having to remove most of it entirely because it became a chain of unused variables.